actix-web/Cargo.toml

```diff
@@ -52,8 +52,12 @@ http2 = ["actix-http/http2"]
 # TLS via OpenSSL
 openssl = ["http2", "actix-http/openssl", "actix-tls/accept", "actix-tls/openssl"]
 
-# TLS via Rustls
-rustls = ["http2", "actix-http/rustls", "actix-tls/accept", "actix-tls/rustls"]
+# TLS via Rustls v0.20
+rustls = ["rustls-0_20"]
+# TLS via Rustls v0.20
+rustls-0_20 = ["http2", "actix-http/rustls-0_20", "actix-tls/accept", "actix-tls/rustls-0_20"]
+# TLS via Rustls v0.21
+rustls-0_21 = ["http2", "actix-http/rustls-0_20", "actix-tls/accept", "actix-tls/rustls-0_20"]
```

Comment:
```suggestion
rustls-0_21 = ["http2", "actix-http/rustls-0_21", "actix-tls/accept", "actix-tls/rustls-0_21"]
```

Reply:
something like this was bound to happen, good catch